examples/gadgets/snapshot/process/process.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/cilium/ebpf/rlimit"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/snapshot/process/tracer"
+)
+
+func main() {
+	if err := rlimit.RemoveMemlock(); err != nil {
```

Comment:
As in other examples:

```suggestion
        // In some kernel versions it's needed to bump the rlimits to
	// use run BPF programs.
	if err := rlimit.RemoveMemlock(); err != nil {
```